bgpd/bgp_vty.c

```diff
@@ -12298,7 +12298,7 @@ ALIAS_HIDDEN(
 
 DEFUN (no_bgp_redistribute_ipv4_ospf,
        no_bgp_redistribute_ipv4_ospf_cmd,
-       "no redistribute <ospf|table> (1-65535) [metric (0-4294967295)] [route-map WORD]",
+       "no redistribute <ospf|table> (1-65535) [{metric (0-4294967295)}|{route-map WORD}]",
```

Comment:
Shouldn't this be ` [{metric (0-4294967295)|route-map WORD}]`?

Reply:
both are optional and in any order, right?

Reply:
This isn't how `{}` works. I just tested your PR:
```
debian(config-router-af)# no redistribute ospf 1 ?
  <cr>
  metric     Metric for redistributed routes
  route-map  Route map reference
debian(config-router-af)# no redistribute ospf 1 route-map RMAP ?
  <cr>
```

Reply:
fixed